### ISSUE TITLE
installation fails at install.php?owa_action=base.installCheckEnv

### ISSUE THREAD

**Issue by sspence65** [padams/Open-Web-Analytics#234]
After filling out the database access form, and clicking Continue:

The www._domainname_.com page isn’t working

www._domainname_.com is currently unable to handle this request.
HTTP ERROR 500

**Comment by sspence65** [comment]
Renamed the dist config file, and filled with same info, that worked. Why did the form not work?

**Comment by padams** [comment]
likely a permissions issue.

**Comment by DOAADI** [comment]
sspence65, what did you mean when you said you renamed the "dist config file"?

I am having a similar problem, where clicking "get started" just brings up a blank screen with the path "domain/owa/install.php?owa_action=base.installCheckEnv"

I've set all permissions to maximum and still no progress.

**Comment by DOAADI** [comment]
Incidentally, trying to do a Wordpress install just throws a fatal error when I try to activate the plugin.

**Comment by Chryseus** [comment]
Same problem, error log has this at the bottom:

> PHP message: PHP Warning:  chmod(): No such file or directory in /var/www/public_html/owa

Also a lot of warnings about fwrite() parameter 1 being a boolean.
Tried mucking around with permissions but no luck.

**Comment by DOAADI** [comment]
Mine was both with Edge and IE.

**Comment by peter-snr** [comment]
Ubuntu 18.04 - Apache2 - MySQL

Open-Web-Analytics-1.6.2.tar.gz

Path to owa:
/var/www/html/owa
Permissions: chown -R www-data:www-data  /var/www/html/owa

MySQL:
database: owa
user created with 'GRANT ALL on owa.* ....

After install I can see all tables.

But.... get this error in apache error_log
[Mon Dec 23 19:37:53.947026 2019] [php7:notice] [pid 20025] [client 172.40.0.80:36826] PHP Notice:  Undefined variable: page_title in /var/www/html/owa/modules/base/templates/wrapper_default.tpl on line 6
[Mon Dec 23 19:37:53.947147 2019] [php7:warn] [pid 20025] [client 172.40.0.80:36826] PHP Warning:  Use of
undefined constant OWA_ERROR_HANDLER - assumed 'OWA_ERROR_HANDLER' (this will throw an Error in a future version of PHP) in /var/www/html/owa/modules/base/templates/config_dom.tpl on line 15
[Mon Dec 23 19:37:54.027635 2019] [php7:warn] [pid 20026] [client 172.40.0.80:36828] PHP Warning:  Use of
undefined constant OWA_ERROR_HANDLER - assumed 'OWA_ERROR_HANDLER' (this will throw an Error in a future version of PHP) in /var/www/html/owa/modules/base/templates/config_dom.tpl on line 15
[Mon Dec 23 19:37:56.100467 2019] [php7:warn] [pid 13652] [client 172.40.0.80:36846] PHP Warning:  Use of
undefined constant OWA_ERROR_HANDLER - assumed 'OWA_ERROR_HANDLER' (this will throw an Error in a future version of PHP) in /var/www/html/owa/modules/base/templates/config_dom.tpl on line 15, referer: http://jupiter.prism.net/owa/install.php
[Mon Dec 23 19:38:12.253079 2019] [php7:notice] [pid 13737] [client 172.40.0.80:36984] PHP Notice:  Undefined variable: page_title in /var/www/html/owa/modules/base/templates/wrapper_default.tpl on line 6
[Mon Dec 23 19:38:12.253185 2019] [php7:warn] [pid 13737] [client 172.40.0.80:36984] PHP Warning:  Use of
undefined constant OWA_ERROR_HANDLER - assumed 'OWA_ERROR_HANDLER' (this will throw an Error in a future version of PHP) in /var/www/html/owa/modules/base/templates/config_dom.tpl on line 15
[Mon Dec 23 19:38:12.334493 2019] [php7:warn] [pid 13654] [client 172.40.0.80:36986] PHP Warning:  Use of
undefined constant OWA_ERROR_HANDLER - assumed 'OWA_ERROR_HANDLER' (this will throw an Error in a future version of PHP) in /var/www/html/owa/modules/base/templates/config_dom.tpl on line 15
[Mon Dec 23 19:38:53.387126 2019] [php7:notice] [pid 13653] [client 172.40.0.80:37332] PHP Notice:  Undefined variable: page_title in /var/www/html/owa/modules/base/templates/wrapper_default.tpl on line 6
[Mon Dec 23 19:38:53.387336 2019] [php7:warn] [pid 13653] [client 172.40.0.80:37332] PHP Warning:  Use of
undefined constant OWA_ERROR_HANDLER - assumed 'OWA_ERROR_HANDLER' (this will throw an Error in a future version of PHP) in /var/www/html/owa/modules/base/templates/config_dom.tpl on line 15
[Mon Dec 23 19:38:53.466233 2019] [php7:warn] [pid 20024] [client 172.40.0.80:37334] PHP Warning:  Use of
undefined constant OWA_ERROR_HANDLER - assumed 'OWA_ERROR_HANDLER' (this will throw an Error in a future version of PHP) in /var/www/html/owa/modules/base/templates/config_dom.tpl on line 15

Any ideas would be appreciated.  I followed these instructions:
https://github.com/padams/Open-Web-Analytics/wiki/Installation#Stand_Alone_Installation

**Comment by padams** [comment]
I’ll look into that but In the meantime Is the error handler commented out in your config file? If so, uncomment it and set the value to “production” (or “development” if you want an error log).

**Comment by peter-snr** [comment]
Ok on it.

**Comment by peter-snr** [comment]
Hi, I decided to start afresh.

Dropped the owa database and recreated.
Deleted /var/www/html/owa
Fresh untar to same dir.
Updated permissions for www-data:www-data
Also updated default apache config to allow .htaccess files (was missing)
Then went to http://myhost.domain/owa/install.php and ran through the installer.

Now it is successful.
Conclusion: Not decisive, perhaps the .htaccess or perhaps I made a mistake the first time.
